ONSdigital/florence
# added ID to handleSearchInput

### What

Users could not search for datasets by ID on the "Select a dataset" page. As such, the `handleSearchInput` function has been updated to also check for ID. Currently it just filters based on the `title` property

### How to review

- Check the code works and is clear in its intention
- Go to the "Select a dataset" page and try to filter based on a dataset ID

### Who can review

Anyone but me
